frontend/src/components/elements/VegaLiteChart/VegaLiteChart.tsx

```diff
@@ -68,7 +68,13 @@ class VegaLiteChart extends React.PureComponent<PropsWithHeight, State> {
   /**
    * The Vega view object
    */
-  private vegaView: vega.View | undefined
+  private vegaView?: vega.View
```

Comment:
Should the type remain as `vega.View | undefined` since we're setting `vegaView` to `undefined` on line 159?